lib/components/learn-more/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Image, Heading, Button, Grid } from '@chakra-ui/react';
+import Link from 'next/link';
+import React from 'react';
+
+interface LearnMoreProps {
+  imgSrc: string;
+  text: string;
+}
+
+const LearnMore: React.FC<LearnMoreProps> = ({ imgSrc, text }) => {
```

Comment:
Should be called Hero or HeroHeader https://www.optimizely.com/optimization-glossary/hero-image/